Resources/Storage/StorageBase.py

```diff
@@ -365,7 +365,7 @@ def constructURLFromLFN(self, lfn, withWSUrl=False):
     # be removed in the next release
     if voLFN != self.se.vo and voLFN != "SandBox" and voLFN != "Sandbox":
 
-      return S_ERROR('LFN does not follow the DIRAC naming convention %s' % lfn)
+      return S_ERROR('LFN (%s) path must start with VO name ' % lfn)
```

Comment:
```suggestion
      return S_ERROR('LFN (%s) path must start with VO name (%s)' % (lfn, self.se.vo))
```
This might also highlight if someone has the wrong proxy for users of multiple VOs, or admin proxies without VO (like myself)

Reply:
I knew you were a schizophrenic :-) . Done